File: Tutorials/10 Days of Statistics/Day 4/geometric_distribution1.jl
function neg_bin_dist(n, k, p, q)
    return binomial(n-1, k-1) * p^k * q^(n-k)
end

str = readline()
n = parse(Int64, readline())
strvec = split(str, ' ')
intvec = map(x -> parse(Int64, x), strvec)
a = intvec[1]
b = intvec[2]

p = a / b
q = 1 - p

println(round(neg_bin_dist(n, 1, p, q), 3))

File: Tutorials/10 Days of Statistics/Day 4/geometric_distribution1.py
# Enter your code here 
#!/bin/python3
import sys
from math import factorial as fac


def binomial(x, y):
    try:
        return fac(x) // fac(y) // fac(x-y)
    except ValueError:
        return 0

def neg_bin_dist(n, k, p, q):
    return binomial(n-1, k-1) * p**k * q**(n-k)

arr = [int(arr_i) for arr_i in input().strip().split(' ')]
n = int(input())
a = arr[0]
b = arr[1]

p = a / b
q = 1 - p

print(round(neg_bin_dist(n, 1, p, q), 3))

